package/gluon-mesh-babel/files/etc/hotplug.d/iface/90-gluon-babel-brclient-no-igmp-snooping

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+case "$ACTION"
+in
+	ifup)
+	echo 1 > /sys/devices/virtual/net/br-client/bridge/multicast_snooping
```

Comment:
What, why did nobody tell me? (ok, now you did :D)

Can you point me to some ticket?